python_scripts/02_numerical_pipeline_hands_on.py

```diff
@@ -141,10 +158,14 @@
 # %% [markdown]
 # ## Train-test split the dataset
 #
-# In the previous notebook, we loaded separately two datasets: a training and a
-# testing dataset. We mentioned that scikit-learn provides an helper function
-# `sklearn.model_selection.train_test_split` allowing to do this split.
-# Here, we will use this tool instead of loading some new data.
+# In the previous notebook, we loaded two separate datasets: a training
+# one and a testing one. However, as mentioned earlier, having separate
+# datasets like that is unusual ; most of the time, we will need to
+# create them from a single dataset we have.
```

Comment:
It is weird to finish a sentence with `we have` :)

Reply:
Rephrase I must